build.gradle

```diff
@@ -97,6 +97,9 @@ dependencies {
     compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
     compile group: 'org.codehaus.janino', name: 'janino', version: '3.0.7'
     compile group: 'com.github.salomonbrys.kodein', name: 'kodein', version: '4.1.0'
+    testCompile 'junit:junit:4.12'
```

Comment:
I tried junit5, but it looked very complicated to add with gradle... so I picked the good old junit4...

Reply:
bon choix, de toute façon JUnit5 ne nous apporte rien :)  
Au pire on pourra passer à TestNG, mais nous n'en sommes pas encore là.

Reply:
Haha, tu es la première personne qui me parle de testng. J'avais essayé de pousser ce framework il y a bien longtemps (avant les annotations de junit) parcequ'il me semblait bien meilleur, notamment avec sa notion d'ordre. Mais junit était déjà le standard! Et du coup j'ai un peu arrêté de suivre le dev de testng !